### ISSUE TITLE
Trigger event grid event at stages of long trainings

### ISSUE THREAD

**Issue by SamVanhoutte** [arcus-azure/arcus.azureml#48]
Check for callback possibilities (like with EarlyStopping , TensorBoards, etc) for Machine Learning (sklearn) and Deep Learning (Keras), that can be added to a fitting job and that can publish a custom event to event grid, based on the configuration.

- Configuration: percentage of completion
- Data to publish: accuracy/metrics of the moment of event triggering, current job/experiment that is running, duration of job, etc

**Comment by SamVanhoutte** [comment]
Wondering if this is still relevant, considering the out of the box integration for run events in AzureML and Event Grid.
